static/panes/opt-pipeline.ts

```diff
@@ -229,12 +229,14 @@ export class OptPipeline extends MonacoPane<monaco.editor.IStandaloneDiffEditor,
 
     updateButtons() {
         if (!this.compiler || !this.compiler.optPipeline) return;
-        const {supportedOptions, supportedFilters} = this.compiler.optPipeline;
```

Comment:
Remarkably, it appears this line never worked.

Reply:
I'm confused why this didn't/doesn't work 🤔

Reply:
My bad!  TIL about javascript destructuring. optPipeline is just empty for most compilers, so in the cases I stepped through supportedOptions|Filters were undefined.   I'll fix that back.

Reply:
Ah I see, thanks for checking!